src/test/java/com/puppycrawl/tools/checkstyle/filters/SuppressionPatchXpathFilterTest.java

```diff
@@ -149,4 +149,15 @@ public void testOuterTypeNumber() throws Exception {
         testByConfig(
                 "OuterTypeNumber/context/defaultContextConfig.xml");
     }
+
+    @Test
+    @Ignore("until https://github.com/checkstyle/patch-filters/issues/157")
```

Comment:
Why is it ignored?

Reply:
https://github.com/checkstyle/patch-filters/pull/158#discussion_r460690753